reconcile/aus/ocm_upgrade_scheduler_org.py

```diff
@@ -26,18 +20,10 @@ def name(self) -> str:
         return QONTRACT_INTEGRATION
 
     def get_ocm_env_upgrade_specs(
-        self, ocm_env: OCMEnvironment, org_ids: Optional[set[str]]
+        self, ocm_env: OCMEnvironment
     ) -> dict[str, OrganizationUpgradeSpec]:
         # query all OCM organizations from app-interface and filter by and orgs
-        organizations = [
-            org
-            for org in aus_organizations_query(
-                query_func=gql.get_api().query
-            ).organizations
-            or []
-            if org.environment.name == ocm_env.name
-            and (org_ids is None or org.org_id in org_ids)
-        ]
+        organizations = self.get_orgs_for_environment(ocm_env)
```

Comment:
nit: the comment above could be fixed (or removed)

Reply:
done